src/main/java/org/usfirst/frc/team1294/robot/commands/ClimbRope.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc.team1294.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import org.usfirst.frc.team1294.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ClimbRope extends Command {
+
+  public ClimbRope() {
+    super("Drive motor");
+    requires(Robot.climbingSubsystem);
+  }
+
+  @Override
+  protected void initialize() {
+
+  }
+
+  @Override
+  protected void execute() {
+    XboxController joystick = Robot.oi.getJoystick();
```

Comment:
Well, is it better to `getJoystick()` and then access the various inputs, or make accessors for them in oi. If we decide on the latter, `MeccanumDriveCommand` would also have to be changed, as it does something similar.